lib/xctest_list.rb

```diff
@@ -35,48 +35,50 @@ def self.binary_path(xctest_bundle_path)
   end
 
   # find the Objective-C symbols in the bundle's binary
-  def self.objc_tests(xctest_bundle_path)
+  def self.objc_tests(xctest_bundle_path, test_prefix: "test")
     tests = []
     objc_symbols_command_output_tempfile = Tempfile.new(File.basename(xctest_bundle_path) + "objc")
     system("nm -U '#{binary_path(xctest_bundle_path)}' > '#{objc_symbols_command_output_tempfile.path}'")
     tests = []
     File.foreach(objc_symbols_command_output_tempfile.path) do |line|
-      if / t -\[(?<testclass>\w+) (?<testmethod>test\w*)\]/ =~ line
-        tests << "#{testclass}/#{testmethod}"
+      regex = / t -\[(?<testclass>\w+) (?<testmethod>#{test_prefix}\w*)\]/
```

Comment:
So, the local variable assignment to `testmethod` will not work if there is variable interpolation in the regular expression: https://sadique.io/blog/2013/10/01/ruby-named-capture-groups-and-local-variables/

Reply:
Thank you *so* much - I was struggling to find the reason why this was happening, but all of my googling turned up nothing.